go/oasis-node/cmd/common/grpc/grpc.go

```diff
@@ -75,15 +68,10 @@ func NewServerLocal(installWrapper bool) (*cmnGrpc.Server, error) {
 	if dataDir == "" {
```

Comment:
Can this also be removed?

Reply:
Yeah I'll look into removing this in a follow up as this was already merged. I don't see a reason why the datadir was required here specifically - even before the change in this PR.

Reply:
> I don't see a reason why the datadir was required here specifically - even before the change in this PR.

To construct the path, I guess.

Reply:
Yeah but this didn't change with the PR. It still uses the same data dir config value to construct it (unless overriden).